kratos/spaces/ublas_space.h

```diff
@@ -399,41 +376,21 @@ class UblasSpace
 
     static void InplaceMult(VectorType& rX, const double A)
     {
+        if (A == 1.00) {
```

Comment:
i wrote the original code, so if there is an error is mine ... but i am now quesitoning myself if we should do A == static_cast<double>(1)